### PR TITLE
Add bundle-icons to fake data cards

### LINKED ISSUE #89: If it's a bundle, repeat the icon (for now)

### PR DESCRIPTION
**Pull request by Lukewh**
## Done

- Update bundle cards to have double icons

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Pull the branch
- Visit http://0.0.0.0:8045/store
- Some cards should have double icons - these are bundles

## Issue / Card

Fixes #89

### CODE REVIEW

**Comment by solazio** [comment]
they are supposed to have 3 icons, see [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e7b5d65c88bbd8cd16bab00)

File: static/sass/_pattern_p-card.scss [hunk 1]
```diff
@@ -1,29 +1,4 @@
 @mixin p-charmhub-card {
-  .p-card--bundle {
```
Comment (solazio) [inline static/sass/_pattern_p-card.scss:2]: removing this css I think messes up the styling for the bundle icon on the bundle details page.

Comment (Lukewh) [reply static/sass/_pattern_p-card.scss:2]: Hmm, I'll check that when get to the details page - not sure the details page should be borrowing styles from the card pattern!

**Comment by Lukewh** [comment]
> they are supposed to have 3 icons, see [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e7b5d65c88bbd8cd16bab00)

Updated, thanks :)!

File: static/sass/_pattern_p-card.scss (hunk 1 above)
Comment (Lukewh) [reply static/sass/_pattern_p-card.scss:2]: You are 100% right - it's for the full bundle cards, not the homepage cards. Readded

**Pull request merged**